samples/snippets/src/main/java/aiplatform/ExportFeatureValuesSample.java

```diff
@@ -0,0 +1,99 @@
+ * https://cloud.google.com/vertex-ai/docs/featurestore/setup before running 
+ * the code snippet
+ */
+
+package aiplatform;
+
+// [START aiplatform_export_feature_values_sample]
+
+import com.google.api.gax.longrunning.OperationFuture;
+import com.google.cloud.aiplatform.v1.BigQueryDestination;
+import com.google.cloud.aiplatform.v1.EntityTypeName;
+import com.google.cloud.aiplatform.v1.ExportFeatureValuesOperationMetadata;
+import com.google.cloud.aiplatform.v1.ExportFeatureValuesRequest;
+import com.google.cloud.aiplatform.v1.ExportFeatureValuesRequest.FullExport;
+import com.google.cloud.aiplatform.v1.ExportFeatureValuesResponse;
+import com.google.cloud.aiplatform.v1.FeatureSelector;
+import com.google.cloud.aiplatform.v1.FeatureValueDestination;
+import com.google.cloud.aiplatform.v1.FeaturestoreServiceClient;
+import com.google.cloud.aiplatform.v1.FeaturestoreServiceSettings;
+import com.google.cloud.aiplatform.v1.IdMatcher;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+public class ExportFeatureValuesSample {
+
+  public static void main(String[] args)
+      throws IOException, InterruptedException, ExecutionException, TimeoutException {
+    // TODO(developer): Replace these variables before running the sample.
+    String project = "YOUR_PROJECT_ID";
+    String featurestoreId = "YOUR_FEATURESTORE_ID";
+    String entityTypeId = "YOUR_ENTITY_TYPE_ID";
+    String destinationTableUri = "YOUR_DESTINATION_TABLE_URI";
+    String location = "us-central1";
+    String endpoint = "us-central1-aiplatform.googleapis.com:443";
+    int timeout = 300;
+    exportFeatureValuesSample(project, featurestoreId, entityTypeId, destinationTableUri, location,
+        endpoint, timeout);
+  }
+
+  static void exportFeatureValuesSample(String project, String featurestoreId, String entityTypeId,
+      String destinationTableUri, String location, String endpoint, int timeout)
+      throws IOException, InterruptedException, ExecutionException, TimeoutException {
+    FeaturestoreServiceSettings featurestoreServiceSettings =
+        FeaturestoreServiceSettings.newBuilder().setEndpoint(endpoint).build();
+
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the "close" method on the client to safely clean up any remaining background resources.
+    try (FeaturestoreServiceClient featurestoreServiceClient =
+        FeaturestoreServiceClient.create(featurestoreServiceSettings)) {
+
+      List<String> ids = new ArrayList<>();
+      ids.add("*");
+      FeatureSelector featureSelector = FeatureSelector.newBuilder()
+          .setIdMatcher(IdMatcher.newBuilder().addAllIds(ids).build()).build();
```

Comment:
nit: inline something like `addAllIds(ArrayList.of()...)`, or make a class variable if intended for developer to update.

Reply:
ok, will add inline as below,
```suggestion
          .setIdMatcher(IdMatcher.newBuilder().addAllIds(Arrays.asList("title","genres","average_rating")).build()).build();
```

Reply:
I feel it's better to make it one of the variables the developer has to update and pass it down here.

Reply:
updated code to pass the feature selector ids as a list object